agent/acs/model/api/api-2.json

```diff
@@ -300,6 +300,13 @@
       }
     },
 
+    "EFSVolumeConfiguration": {
+      "type":"structure",
+      "members":{
+        "filesystemId":{"shape":"String"},
```

Comment:
In our PRFAQ, this attribute is defined as **filesystem**, ACS will keep the consistency and use **filesystem**. Just a reminder that we may want to wait for ACS model finalized and regenerate the file again.

Reply:
yep, the exact shape of this is subject to change, we'll keep this code in a feature branch until the ACS model is finalized